res/values/config.xml

```diff
@@ -560,4 +560,8 @@
 
     <!-- Whether the given language capitalizes nouns -->
     <bool name="language_capitalizes_nouns">false</bool>
+
+    <!-- Smart Charging -->
+    <bool name="config_supportSmartCharging">false</bool>
```

Comment:
On crDroid 7 it was `config_smartChargingAvailable` and it still present in most device overlays. So, it's better to leave old naming.